<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainVLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tabTextOutput">
        <attribute name="title">
         <string>Text output</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="textOutputEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabVisualOutput">
        <attribute name="title">
         <string>Visual output</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="visualOutputLabel">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string>Das ist ein Test!!!!</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="hSliderDrawRange">
           <property name="value">
            <number>99</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Video Player</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="textEdit_video_player"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Playback</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="pBtn_playback_play">
                <property name="text">
                 <string>Play</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBtn_playback_pause">
                <property name="text">
                 <string>Pause</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBtn_playback_get">
                <property name="text">
                 <string>Get Status</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Frame number</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="pBtn_frameno_get">
                <property name="text">
                 <string>Get current frame no</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sBox_frameno">
                <property name="maximum">
                 <number>10000000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBtn_frameno_set">
                <property name="text">
                 <string>Set current frame no</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pBtnGetCurrent">
         <property name="text">
          <string>GET Current</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBtnGetCurrentDetails">
         <property name="text">
          <string>GET Current Details</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBtnTest">
         <property name="text">
          <string>Test Button</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
